### ISSUE TITLE
Accessibility of Tabs with manual activation

### ISSUE THREAD

**Issue by patrickdemers** [w3c/wcag#3220]
I have few questions about tabs with manual activation pattern. We have our own implementation of the pattern (we don't follow each guidelines of the APG pattern) and I want to know if we fail under 1.3.2 Meaningful sequence and 2.4.3 Focus order

- Why do the user needs to move the focus with the left and right arrow instead of the tab key?
- Is it OK to add a tabindex=0 on the tabpanel?

**Comment by srinivasuchakravarthula** [comment]
For tabpanel, I do think navigating through left/right arrow keys (for
horizontal tabs) would be helpful so that user can use "tab" key to move to
next focusable component. Let's say if there are 20 tabs, user is not
forced to browse through all of those tabs.
Regards,
Srinivasu

Srinivasu Chakravarthula, CPWA, DHS TT
Phone: +91 99008 10881
Website <http://www.srinivasu.org> | LinkedIn
<http://linkedin.com/in/srinivasuc> | Facebook
<http://facebook.com/srinivasu>

Director of Product Accessibility, Freshworks Inc
Freelance Accessibility Consultant

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do so.
